lerna-package-lock.json

```diff
@@ -4545,7 +4545,7 @@
 		"@microsoft/tsdoc": {
 			"version": "0.12.19",
 			"resolved": "https://registry.npmjs.org/@microsoft/tsdoc/-/tsdoc-0.12.19.tgz",
-			"integrity": "sha512-IpgPxHrNxZiMNUSXqR1l/gePKPkfAmIKoDRP9hp7OwjU29ZR8WCJsOJ8iBKgw0Qk+pFwR+8Y1cy8ImLY6e9m4A==",
```

Comment:
Remove lerna-package-lock.json and package-lock.json from this change -- you'll typically only want to commit them if you're making a change to an installed dependency (adding, removing, changing version, etc.)